setup.py

```diff
@@ -42,7 +42,8 @@
     'ftfy>=4.4.3',
     'marshmallow>=3.0.0,<4.0.0',
     'pycountry>=18.12.8',
-    "uritemplate>=3.0.1",
+    'uritemplate>=3.0.1',
+    'werkzeug>=1.0.0',
```

Comment:
We should be paranoid about closing these open-ended dependencies now ;)

```suggestion
    'uritemplate>=3.0.1,<4.0.0',
    'werkzeug>=1.0.0,<2.0.0',
```